### ISSUE TITLE
templates per techs

### ISSUE THREAD

**Issue by spuhlerd** [santiago-sanitation-systems/Santiago.jl#41]
Analyse all temlates to check
possible function:
tech_templates(allSys, tech)

where to store this info in order to be available for the UX in a easy way...

**Comment by scheidan** [comment]
Currently we have:
```Julia
templates_per_tech(systems::Array{System})
techs_per_template(systems::Array{System})
```

**Comment by scheidan** [comment]
closed by 7308caf7980fc994ab7562ddfe4bf7d36e3700b9
